src/cmd/flux-kvs.c

```diff
@@ -250,31 +250,38 @@ int main (int argc, char *argv[])
     return (exitval);
 }
 
-static void output_key_json_object (const char *key, json_object *o)
+static void output_key_json_object (const char *key, json_t *o)
 {
+    char *s;
     if (key)
         printf ("%s = ", key);
 
-    switch (json_object_get_type (o)) {
-    case json_type_null:
+    switch (json_typeof (o)) {
+    case JSON_NULL:
         printf ("nil\n");
         break;
-    case json_type_boolean:
-        printf ("%s\n", json_object_get_boolean (o) ? "true" : "false");
+    case JSON_TRUE:
+        printf ("true\n");
         break;
-    case json_type_double:
-        printf ("%f\n", json_object_get_double (o));
+    case JSON_FALSE:
+        printf ("false\n");
         break;
-    case json_type_int:
-        printf ("%d\n", json_object_get_int (o));
+    case JSON_REAL:
+        printf ("%f\n", json_real_value (o));
         break;
-    case json_type_string:
-        printf ("%s\n", json_object_get_string (o));
+    case JSON_INTEGER:
+        printf ("%lld\n", (long long)json_integer_value (o));
```

Comment:
Yeah, I personally find @grondo's preference much easier to remember. I always had to go back to a book to remember those PRI macros.